dpnp/tests/test_sycl_queue.py

```diff
@@ -2135,11 +1416,358 @@ def test_ix(device_0, device_1):
     assert_sycl_queue_equal(ixgrid[1].sycl_queue, x1.sycl_queue)
 
 
-@pytest.mark.parametrize(
-    "device", valid_dev, ids=[dev.filter_string for dev in valid_dev]
-)
+@pytest.mark.parametrize("device", valid_dev, ids=dev_ids)
 def test_choose(device):
     chc = dpnp.arange(5, dtype="i4", device=device)
     inds = dpnp.array([0, 1, 3], dtype="i4", device=device)
     result = dpnp.choose(inds, chc)
     assert_sycl_queue_equal(result.sycl_queue, chc.sycl_queue)
+
+
+class TestLinAlgebra:
```

Comment:
It seems we can parameterize the class instead of `@pytest.mark.parametrize("device", valid_dev, ids=dev_ids)` per every method.